Hello microverse setup - Resubmission

mohamedgamaleldin999999 opened the pull request:
This is a new pull request after I worked on the changes that were required from me. I fixed all linting mistakes. I created a new pull request as advised. I also created a new branch with a more descriptive name. I am also trying to make this PR with a more descriptive heading and summary. I have worked on the requested changes on the README file. I added the sections that I had removed earlier. That includes: Built With, Tech Stack, Key Features, Prerequisites, and Future Features.
here is a link to the first pull request that I used in my first submission. https://github.com/mohamedgamaleldin999999/The-Jimmy-Project/pull/2 .
I am aware of the fact that I should not use a new PR for my new submission. I opened a new PR because I was requested to do so by my first code review.
Thank you.

didierganthier commented on README.md:
```diff
@@ -1 +1,132 @@
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
```
Nice job so far 😎 but as required by the previous reviewer [here](https://github.com/mohamedgamaleldin999999/The-Jimmy-Project/pull/2#discussion_r1176901782) make sure that all the sections here are referenced below, remember only the `FAQ` is optional, 🤷🏽  if you have any doubts or questions about a section, feel free to tag me here or ask around, some sections can be removed such as the `Live Demo` 🚀  which is only applicable once you have a deployed version but some others are necessary. 🤓

mohamedgamaleldin999999 replied:
@didierganthier 
Hello, My friend. I have a question. Is it ok to ask you to manually have a look at my repo before I submit my third submission since it will be the last one. if not, could you please confirm to me if "add javascript" could be considered as valid Future Feature? 
thank you.

didierganthier replied:
Hello @mohamedgamaleldin999999 , yes it's perfectly fine to ask me to take a look and yes this would be considered a valid Future Feature 👍🏽

mohamedgamaleldin999999 replied:
Ok. I think I am ready to submit a third submission now. Could you please have a quick look? Thank you.

mohamedgamaleldin999999 replied:
@didierganthier

didierganthier replied:
wait

didierganthier replied:
You must add it [here](https://github.com/mohamedgamaleldin999999/The-Jimmy-Project/pull/3#issue-1684019391) in the PR description.

didierganthier replied:
The README looks good to me 👍🏽

mohamedgamaleldin999999 replied:
Thanks a lot @didierganthier . how about now?

didierganthier replied:
Looks great @mohamedgamaleldin999999

didierganthier reviewed (changes requested):
Hi @mohamedgamaleldin999999 ,

🎉 Great job on addressing the feedback from the previous review! You've made some solid progress in implementing the requested changes. However, there are still a few areas that need some attention before we can move on to the next project.

<img src="https://media.giphy.com/media/e56n4Z0mmkxZS/giphy.gif" />

## Highlights
- Descriptive Pull Request ✔️ 
- Good Linters Setup ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- [ ] You're making great progress 💯 but make sure that, if you're submitting a new PR after the first one was reviewed, to add the old PR link to the description. So I suggest you edit your description to include this [link](https://github.com/mohamedgamaleldin999999/The-Jimmy-Project/pull/2) so that any reviewer can see the previous reviews. 📖 

- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- N/A

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

mohamedgamaleldin999999 commented:
Hello.
I have worked on the requested changes on the README file. I added the sections that I had removed earlier. That includes: Built With, Tech Stack, Key Features, Prerequisites, and Future Features.
here is a link to the first pull request that I used in my first submission. https://github.com/mohamedgamaleldin999999/The-Jimmy-Project/pull/2 .
I am aware of the fact that I should not use a new PR for my new submission. I opened a new PR because I was requested to do so by my first code review.
Thank you.

V-Blaze reviewed (approved):
Hi @mohamedgamaleldin999999  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
![giphy](https://user-images.githubusercontent.com/107632780/229644741-7dea3027-bf3f-4416-855b-46095e3bf6d5.gif)

## Highlights

- Your pull request summary is very descriptive💯
-  Readme file looks proffessional👍
-  Excellent communication with the reviewer🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

shahadat3669 reviewed (commented):
Hi @mohamedgamaleldin999999 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

<img src="https://media1.giphy.com/media/jmqAUNFXAElgafPK0g/giphy.gif"/>

## Invalid Code Review Request
It appears that the project you have submitted is invalid. Please make sure to submit the correct repository based on project requirements.

## 🔴🚫Attention🚫🔴 

📢 Your Code Review Request will be marked as invalid🚫 in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

## Have any doubt ❓

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

<table width="100%" align="center">
  <tr>
    <td colspan="4" align="center">
      <strong>Contact me: Shahadat Hossain</strong>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="https://github.com/shahadat3669">
        <img src="https://img.shields.io/badge/github-%23121011.svg?style=for-the-badge&logo=github&logoColor=white" alt="GitHub">
      </a>
    </td>
    <td align="center">
      <a href="https://www.linkedin.com/in/shahadat-cseng/">
        <img src="https://img.shields.io/badge/linkedin-%230077B5.svg?style=for-the-badge&logo=linkedin&logoColor=white" alt="LinkedIn">
      </a>
    </td>
    <td align="center">
      <a href="https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V">
        <img src="https://img.shields.io/badge/Slack-4A154B?style=for-the-badge&logo=slack&logoColor=white" alt="Slack">
      </a>
    </td>
    <td align="center">
      <a href="mailto:shahadat3669@gmail.com?subject=Microverse%20Code%20Review">
        <img src="https://img.shields.io/badge/Gmail-D14836?style=for-the-badge&logo=gmail&logoColor=white" alt="Email">
      </a>
    </td>
  </tr>
</table>

## ⚠️ WARNING ⚠️

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_**📢💡Invalid Code Review Request **does not count** into the code reviews limit.💡**_

topeogunleye commented:
> Hi @mohamedgamaleldin999999
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ![](https://camo.githubusercontent.com/88e0e36998e5cc5c01d0266f6819ba163f8fdc75094d809a709639409e54e326/68747470733a2f2f6d65646961312e67697068792e636f6d2f6d656469612f6a6d7141554e465841456c676166504b30672f67697068792e676966) [ ![68747470733a2f2f6d65646961312e67697068792e636f6d2f6d656469612f6a6d7141554e465841456c676166504b30672f67697068792e676966](https://camo.githubusercontent.com/88e0e36998e5cc5c01d0266f6819ba163f8fdc75094d809a709639409e54e326/68747470733a2f2f6d65646961312e67697068792e636f6d2f6d656469612f6a6d7141554e465841456c676166504b30672f67697068792e676966) ](https://camo.githubusercontent.com/88e0e36998e5cc5c01d0266f6819ba163f8fdc75094d809a709639409e54e326/68747470733a2f2f6d65646961312e67697068792e636f6d2f6d656469612f6a6d7141554e465841456c676166504b30672f67697068792e676966) [ ](https://camo.githubusercontent.com/88e0e36998e5cc5c01d0266f6819ba163f8fdc75094d809a709639409e54e326/68747470733a2f2f6d65646961312e67697068792e636f6d2f6d656469612f6a6d7141554e465841456c676166504b30672f67697068792e676966)
> 
> ## Invalid Code Review Request
> It appears that the project you have submitted is invalid. Please make sure to submit the correct repository based on project requirements.
> 
> ## 🔴🚫Attention🚫🔴
> 📢 Your Code Review Request will be marked as invalid🚫 in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> ## Have any doubt ❓
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **Contact me: Shahadat Hossain**
> [ ![GitHub](https://camo.githubusercontent.com/410d86e43f847d3f6e3027fa6f0c2fb7641d893fa601d863a943eac968c41890/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f6769746875622d2532333132313031312e7376673f7374796c653d666f722d7468652d6261646765266c6f676f3d676974687562266c6f676f436f6c6f723d7768697465) ](https://github.com/shahadat3669)	[ ![LinkedIn](https://camo.githubusercontent.com/29ba59dbf61686238096822c7de916a9b41c40bf362b70e7f2c609551ce8f656/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f6c696e6b6564696e2d2532333030373742352e7376673f7374796c653d666f722d7468652d6261646765266c6f676f3d6c696e6b6564696e266c6f676f436f6c6f723d7768697465) ](https://www.linkedin.com/in/shahadat-cseng/)	[ ![Slack](https://camo.githubusercontent.com/4259e8001faaf7cee8a6cd41e58c905467bfaeae23251d99d1a9c46956d7e713/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f536c61636b2d3441313534423f7374796c653d666f722d7468652d6261646765266c6f676f3d736c61636b266c6f676f436f6c6f723d7768697465) ](https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V)	[ ![Email](https://camo.githubusercontent.com/71a0f4bfcf1f2220e2b1c246ac2ee681c47ee914d1c1f0e27a0e6c9ac2e9f134/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f476d61696c2d4431343833363f7374796c653d666f722d7468652d6261646765266c6f676f3d676d61696c266c6f676f436f6c6f723d7768697465) ](mailto:shahadat3669@gmail.com?subject=Microverse%20Code%20Review)
> ## ⚠️ WARNING ⚠️
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _**📢💡Invalid Code Review Request does not count into the code reviews limit.💡**_
